appcenter-push/scripts/postinstall.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const path = require('path');
+
+const projectDirectory = path.resolve(__dirname, '..', '..', '..');
+const setupFileName = `.${path.sep}node_modules${path.sep}appcenter-push${path.sep}test${path.sep}appCenterPushMock.js`;
+
+// Check if package.json has jest as dependency
+let packageJson = '';
+const packageJsonFile = path.join(`${projectDirectory}`, 'package.json');
```

Comment:
This is a lot of duplicated code, we can share code in the linker scripts package and use it here.